README.md

```diff
@@ -1,2 +1,14 @@
 # redstone-multimeter-fabric
 A Fabric port of NarcolepticFrog's Redstone Multimeter Mod
+
+## The User Interface
+
+The following graphic shows the main components of the Redstone Multimeter user interface. You can add or remove a meter by looking at a block and pressing the `Toggle Meter` keybinding (default `m`). Holding control while placing a meter makes it unmovable (unmovable meters do not render a wireframe cube). You can pause/unpause the meters by pressing the `Pause Meters` keybinding (default `n`). While paused, you can scroll forwards and backwards through time by pressing the `Step Backward` and `Step Forward` keybindings (default `,` and `.`, respectively). Holding control while stepping forwards or backwards jumps 10 game ticks at a time.
+
+![User Interface Overview](https://raw.githubusercontent.com/NarcolepticFrog/RedstoneMultimeter/master/figures/UIOverview.png)
```

Comment:
Unfortunately this image is a bit outdated for RSMM Fabric. The horizontal line looks different now and it is used for many more events besides block movements. The legend is also incorrect regarding the powered state of the meter. In the Litemod, "powered" means different things for different blocks, and the Fabric mod has split this into the "powered" and "active" event types. Ideally the legend would explain that.